lib/tasks/replay_submission.rake

```diff
@@ -0,0 +1,49 @@
+namespace :replay_submission do
+  desc "
+  Replay a submission
+  Usage
+  rake replay_submission:process[<submission_id>]
+  "
+  task :process, [:submission_id] => :environment do |_t, args|
+    if args[:submission_id].nil?
+      puts 'Submission ID is required'
+    else
+      job = Delayed::Job.all.find { |j| j.handler.include?(args[:submission_id]) }
+      if job.nil?
+        puts "No job for submission ID #{args[:submission_id]}"
+      else
+        job.update(run_at: 10.seconds.from_now)
+        puts "Replayed job for submission ID #{args[:submission_id]}"
```

Comment:
Do you mind do add the job id on this message too, please? Maybe at first, it is not important but if the job did process right would be faster to know the job id.